compiler/cli/src/org/jetbrains/kotlin/cli/jvm/compiler/coreApplicationEnvironmentCompat.kt

```diff
@@ -9,7 +9,7 @@ import com.intellij.core.CoreApplicationEnvironment
 import com.intellij.openapi.extensions.ExtensionsArea
 import java.io.File
 
-// BUNCH: 193
+// FIX ME WHEN BUNCH 193 REMOVED
```

Comment:
There are also `// BUNCH 192` already in use. Les's change them as well?

Also I see couple `// BUNCH: as36, 191`. Do you think they can also be modified?

Reply:
> There are also // BUNCH 192 already in use. Les's change them as well?

You have a sharp eye. Thanks, fixed them as well

> Also I see couple // BUNCH: as36, 191. Do you think they can also be modified?

Fixed